### DISCUSSION TITLE
Debugging karma-test-explorer outside a Dev Container

### DISCUSSION THREAD

**Discussion by VIVIsectI** [lucono/karma-test-explorer#55] [Issues / Help] [answered]
Curious if karma-test-explorer can be debugged outside a dev container. My dev environment is a virtual machine and I can't run Docker. I have access to a kubernetes cluster for deploying containers but instructions are sparse. Wonder if anyone's had success debugging the extension's dev container in a kubernetes environment or if it can be debugged locally.

**Comment by lucono** [comment, marked as answer]
@VIVIsectI debugging Karma Test Explorer does not rely on Docker or a containerized environment, and should work in any environment that's got node and VSCode.
